scripts/test

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
-npm test && npx nyc report
+npm run test:unit
+npm run test:ct-unit
+npm start & wait-on http://localhost:3000
+npm run test:e2e
```

Comment:
Is this just for us to run? Maybe it makes sense either way to add `set -e` at the top of the file, so it exits if any of the commands fail?

Reply:
hmm the CI scripts already end the node process with a non zero exit code if any command fails, but we can add set -e

Reply:
I'm not aware of such functionality, are you sure about this?

Reply:
I added `set -e` to all script files, since a successful command down the line can still return a 0 exit code..